scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -25,6 +25,30 @@ label {
   fieldset[disabled] & {
     color: $crm-darker-grayblue;
   }
+
+  &[type="text"] {
+    &::-webkit-input-placeholder {
```

Comment:
apologies, i just realized that this being for the placeholder it requires vendor-specific selectors